src/Uno.Toolkit.Skia.WinUI/SkiaCompact.skia.cs

```diff
@@ -0,0 +1,314 @@
+﻿#nullable enable
+
+using System;
+using System.Runtime.CompilerServices;
+using SkiaSharp;
+using static SkiaSharp.SKImageFilter;
+
+namespace Uno.Toolkit.Skia.WinUI;
+
+internal static class SkiaCompat
```

Comment:
are we making a full transition into "SkiaCompat" or just a few methods?
some commentary could be useful here

Reply:
It was just for a few methods, this is bringing in the whole class from the Uno repo for SkiaCompat, I can clean it up and only include the methods we need